src/filter.js

```diff
@@ -0,0 +1,105 @@
+var Crossfilter = require("crossfilter");
+
+function Filter(){
+  this.crossfilter = new Crossfilter();
+  this.dimensions = {};
+  this.tiles = new Set();
```

Comment:
:ok_hand: just changed this, it was supposed to be a temporary use. Replaced it with a normal JS object.